app/Main.hs

```diff
@@ -26,6 +27,10 @@ shower tag = forever $ waitThen (s " -> " sendFwd) (s " <- " sendRev)
   where
     s d act ms = liftIO (putStrLn $ tag ++ d ++ (show ms)) >> act ms
 
+-- FIXME: This is owned by something unsendable and we should reflect that
+apiClaimed :: Owners (AddrWithUser SockAddr String)
+apiClaimed = Map.singleton "api" $ AddrWithUser (SockAddrCan 12) "relay"
```

Comment:
Hmmm, `(SockAddrCan 12)`, eh? :thinking:

Reply:
I think it's an error if we ever try to send here, but this won't give us anything good.